Gemfile

```
@@ -19,10 +20,14 @@ group :development, :test do
 end
 
 group :development do
-  gem "faker", "~> 2.14"
   gem "letter_opener_web", "~> 2.0"
   gem "listen", "~> 3.1"
+  gem "rubocop-faker"
   gem "spring", "~> 2.0"
   gem "spring-watcher-listen", "~> 2.0.0"
   gem "web-console", "~> 4.2"
 end
+
+group :test do
+  gem "codecov", require: false
```

Same here, can we add a pin for this one? 
```suggestion
  gem "codecov", "~> 0.6.0", require: false
```

---

done